docs/online_config_changes.md

```diff
@@ -0,0 +1,25 @@
+# Online Configuration Updates {#ovms_docs_online_config_changes}
+
+### Updating Configuration File
+OpenVINO Model Server monitors changes to the configuration file and applies required modifications during runtime using two different methods:
+
+1. Automatically, with an interval defined by the parameter `--file_system_poll_wait_seconds`. (introduced in version 2021.1)
+2. On demand, using the [Config Reload API](./model_server_rest_api.md#config-reload). (introduced in version 2021.3)
+
+Configuration reload triggers the following operations:
+
+- new model(s) or [DAG(s)](./dag_scheduler.md) are added to the configuration file, loaded and served.
+- changes to the configured model storage (e.g. new model version is added) are applied. 
+- changes to the configuration of deployed models and [DAGs](./dag_scheduler.md) are applied. 
+- all model versions will be reloaded when there is a change to the model configuration.
+- when a deployed model or [DAG](./dag_scheduler.md) is deleted from `config.json`, it will be unloaded completely from the server after already running inference operations have completed.
```

Comment:
```suggestion
- when a deployed model or [DAG](./dag_scheduler.md) is deleted from `config.json`, it will be unloaded completely from the server after already running inference operations have been completed.
```